tests/Behat/Context/Ui/Admin/ManagingProducts.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Webgriffe\SyliusAkeneoPlugin\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Behat\Mink\Element\NodeElement;
+use Sylius\Behat\Page\Admin\Product\IndexPageInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProducts implements Context
```

Comment:
All context classes should be named with the `Context` suffix. So this should be renamed to `ManagingProductsContext`.